src/views/Home/components/Speakers/SpeakerCard/SpeakerCard.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
+import "./index.scss"
+import { CardProps } from "Props/props"
+import { ReactComponent as ImgOverlay } from "images/speakers/overlay.svg"
+
+const SpeakerCard: React.FC<CardProps> = ({
+  name,
+  title,
+  imgSrc,
+}: CardProps) => (
+  <div className='speaker-card'>
+    <div className='speaker-card__img-container'>
+      <ImgOverlay className='speaker-card__img-overlay' />
+      <img src={imgSrc} alt={name} className='speaker-card__image' />
+    </div>
+    <div className='speaker-card__card-container'>
+      <div className='speaker-card__name'>{name}</div>
+      <div className='speaker-card__title'>
+        {title.split("\n").map(val => (
+          <>
+            {val}
+            <br />
```

Comment:
I'm not the biggest fan of using br tags. Overall it looks good but do you think there's another way we can implement this consistently. I know realistically `<div>` should be able to work but if this works best we can keep it like this.

Reply:
Ah yeah, I decided to add this last minute so this can probably be implemented better. It looks like `<div>`s work pretty well so I'll use that instead.